src/ExternalIcon.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+export const ExternalIcon: React.FC = () => (
```

Comment:
Nit: should probably add this to the Icons list

Reply:
How would you handle different viewbox?

Reply:
There's a `makeIcon` function in there that allows you to pass a different viewBox